readme.md

```diff
@@ -35,6 +35,14 @@ packageJson('@company/package', 'latest').then(json => {
 You can optionally specify a version (e.g. `1.0.0`) or `latest`.  
 If you don't specify a version you'll get the [main entry](http://registry.npmjs.org/pageres/) containing all versions.
 
+The version can also be in the form that is often used in the package.json files:
```

Comment:
I would be more specific about it supporting any semver syntax and linkify node-semver.